caps/types.go

```diff
@@ -83,3 +86,52 @@ func (t *Type) MarshalJSON() ([]byte, error) {
 func (t *Type) UnmarshalJSON(data []byte) error {
 	return json.Unmarshal(data, &t.Name)
 }
+
+// GrantPermissions makes it possible for the package `snapName` to use hardware
```

Comment:
'hardware' is probably not the right word here for this comment when considering pure security capabilities (eg, 'firewall-management' as opposed to 'camera')